style.css

```diff
@@ -15,12 +31,23 @@
   margin: 0;
 }
 
+.colorName { /*found on css-tricks*/
```

Comment:
Excellent!  Thanks for sharing the source.  That way I can go back and look for updates, motivations, and reasons why.